### PR TITLE
config: remove gplus from generation as G+ is not alive anymore

### PR DESCRIPTION
**Pull request by romani**
```
✔ ~/java/github/checkstyle/contribution/releasenotes-builder [remove-gplus L|✔] 
$ ag "plus"
target/checkstyle-checker.xml
662:      <property name="tokens" value="UNARY_PLUS"/>
679:      <property name="tokens" value="PLUS"/>
703:      <property name="tokens" value="PLUS_ASSIGN"/>

src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java
72:        assertFile("twitterGplusBreakingCompatibility.txt", MainProcess.TWITTER_FILENAME);
86:        assertFile("twitterGplusNew.txt", MainProcess.TWITTER_FILENAME);
100:        assertFile("twitterGplusBug.txt", MainProcess.TWITTER_FILENAME);
114:        assertFile("twitterGplusMisc.txt", MainProcess.TWITTER_FILENAME);
128:        assertFile("twitterGplusNew.txt", MainProcess.TWITTER_FILENAME);
147:        assertFile("twitterGplusAll.txt", MainProcess.TWITTER_FILENAME);
175:        assertFile("twitterGplusBreakingCompatibility.txt", MainProcess.TWITTER_FILENAME);

config/intellij-idea-inspections.xml
384:    <inspection_tool class="ConfusingPlusesOrMinusesJS" enabled="true" level="ERROR" enabled_by_default="true" />
2139:    <inspection_tool class="UnaryPlus" enabled="true" level="ERROR" enabled_by_default="true" />

```

### CODE REVIEW

**Comment by rnveach** [comment]
GPlus is optional right? Why just not enable it and leave the code in?

**Comment by romani** [comment]
It will never come back , it is identical to Twitter by content. No reason to keep.

File: releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java [hunk 1]
```diff
@@ -70,7 +70,6 @@ public void testGenerateOnlyBreakingCompatibility() throws Exception {
 
         assertFile("xdocBreakingCompatibility.txt", MainProcess.XDOC_FILENAME);
         assertFile("twitterGplusBreakingCompatibility.txt", MainProcess.TWITTER_FILENAME);
```
Comment (rnveach) [inline releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java]: Will you remove google plus from file name?

Comment (romani) [reply releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java]: I tried to minimize changes .. 
Ok, I will remove

Comment (romani) [reply releasenotes-builder/src/test/java/com/github/checkstyle/templates/TemplateProcessorTest.java]: done

**Pull request merged**